CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
```

Comment:
Ahhhh dang it, I missed this. There needs to be a space after the hash to make it a heading.